Docker Desktop update redesign docs (#12623)
docker/docker.github.io: docker-for-mac/install.md

Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available automatically as delta updates from the previous version.

> When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. Whenever convenient, you can start the download of the update in the background.

Reviewer: ```suggestion
When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can start the download of the update in the background whenever it is convenient for you.
```

> When the download finishes, all you need to do is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.

Reviewer: ```suggestion
After downloading the update, all you need to do is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.
```
